### PR TITLE
Add code file and description

### PR DESCRIPTION
**Pull request by angemariya**
Sorry I worked on the wrong branch before :)

### CODE REVIEW

File: HackerRankIssues/MiniMaxSumTask/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Function Description
+
+Complete the miniMaxSum function in the editor below.
+
+miniMaxSum has the following parameter(s):
+
+    arr: an array of 5 integers
+
+Print
```
Comment (sidhtc510) [inline HackerRankIssues/MiniMaxSumTask/README.md:19]: Shikardos

**Review by sidhtc510 — APPROVED** [review]

File: HackerRankIssues/MiniMaxSumTask/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Given five positive integers, find the minimum and maximum values that can be calculated by summing exactly four of the five integers. Then print the respective minimum and maximum values as a single line of two space-separated long integers.
```
Comment (rasnaut) [inline HackerRankIssues/MiniMaxSumTask/README.md:1]: Please add # to title

**Review by rasnaut — CHANGES_REQUESTED** [review]

**Review by sidhtc510 — APPROVED** [review]
ok

**Review by rasnaut — APPROVED** [review]
All are cool!

**Pull request merged**